user/serializers.py

```diff
@@ -0,0 +1,199 @@
+from rest_framework import serializers
+from user.models import Wallet, Watchlist, Inventory
+from item.models import Currency, Item
+from django.contrib.auth.models import User
+from rest_framework_jwt.settings import api_settings
+
+
+class UserSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = User
+        fields = ('username',)
+
+
+class CurrencySerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Currency
+        fields = ('code', 'name',)
+
+
+class ItemSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Item
+        fields = ('name', 'code',)
+
+
+class WalletCreateSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Wallet
+        fields = ('currency', 'user')
+        read_only_field = ('id',)
+
+    def create(self, validated_data):
+        return Wallet.objects.create(
+            user=validated_data['user'],
+            currency=validated_data['currency'],
+            balance=validated_data['balance']
```

Comment:
**FYI**, i think for better readability in the future it will be great to unpack `validated_data` first
```python
user, currency, balance = (
    validated_data['user'],
    validated_data['currency'],
    validated_data['balance'],
)
return Wallet.object.create(user=user, currency=currency, balance=balance)
```